_includes/styles.html

```diff
@@ -1,17 +1,39 @@
 {% if page.ref == 'homepage' or page.ref == 'dashboard' %}
-    <!-- Allow Bootstrap override -->
-    <link href="{{ site.baseurl }}/assets/dashboard/app/css/app.css" rel="preload" as="style">
-    <link href="{{ site.baseurl }}/assets/dashboard/app/css/app.css" rel="stylesheet">
+<!-- Allow Bootstrap override -->
+<link
+  rel="preload"
+  href="{{ site.baseurl }}/assets/dashboard/_next/static/css/pages/index.css"
+  as="style"
+  crossorigin=""
+/>
+<link
+  rel="stylesheet"
+  href="{{ site.baseurl }}/assets/dashboard/_next/static/css/pages/index.css"
+  crossorigin=""
+  data-n-p=""
+/>
 {% endif %}
 
 <!-- Bootstrap core CSS -->
-<link rel="stylesheet" href="{{ site.baseurl }}/assets/bootstrap-italia/dist/css/bootstrap-italia.min.css">
+<link
+  rel="stylesheet"
+  href="{{ site.baseurl }}/assets/bootstrap-italia/dist/css/bootstrap-italia.min.css"
+/>
```

Comment:
Code reformatting has impacted these lines. We shouldn't see this change. Please reinstate the initial condition.

```suggestion
<link rel="stylesheet" href="{{ site.baseurl }}/assets/bootstrap-italia/dist/css/bootstrap-italia.min.css">
```